src/main/java/frc/team2412/robot/subsystems/DrivebaseSubsystem.java

```diff
@@ -147,7 +148,7 @@ public DrivebaseSubsystem() {
 
 		// LOW verbosity only sends field position, HIGH sends full drive data, MACHINE sends data
 		// viewable by AdvantageScope
-		SwerveDriveTelemetry.verbosity = TelemetryVerbosity.LOW;
+		SwerveDriveTelemetry.verbosity = TelemetryVerbosity.MACHINE;
```

Comment:
Should this also be controlled by debug flag?

Reply:
shouldnt be a problem as long as we merge the logging pr